pkg/cmd/experimental/ipfailover/ipfailover.go

```diff
@@ -163,18 +174,52 @@ func Run(f *clientcmd.Factory, options *ipfailover.IPFailoverConfigCmdOptions, c
 		return err
 	}
 
-	if options.Action.ShouldPrint() {
-		mapper, _ := f.Object(false)
-		return cmdutil.VersionedPrintObject(f.PrintObject, cmd, mapper, options.Action.Out)(list)
-	}
-
 	namespace, _, err := f.DefaultNamespace()
 	if err != nil {
 		return err
 	}
+	_, kClient, err := f.Clients()
+	if err != nil {
+		return fmt.Errorf("error getting client: %v", err)
+	}
+	if err := validateServiceAccount(kClient, namespace, options.ServiceAccount); err != nil {
+		return fmt.Errorf("ipfailover could not be created; %v", err)
+	}
+
+	configList := []runtime.Object{
+		&kapi.ServiceAccount{ObjectMeta: kapi.ObjectMeta{Name: options.ServiceAccount}},
+	}
+
+	list.Items = append(configList, list.Items...)
+
+	if options.Action.ShouldPrint() {
+		mapper, _ := f.Object(false)
+		return cmdutil.VersionedPrintObject(f.PrintObject, cmd, mapper, options.Action.Out)(list)
+	}
 
 	if errs := options.Action.WithMessage(fmt.Sprintf("Creating IP failover %s", name), "created").Run(list, namespace); len(errs) > 0 {
 		return cmdutil.ErrExit
 	}
 	return nil
 }
+
+func validateServiceAccount(client *kclient.Client, ns string, serviceAccount string) error {
+
+	sccList, err := client.SecurityContextConstraints().List(kapi.ListOptions{})
+	if err != nil {
+		if !errors.IsUnauthorized(err) {
+			return fmt.Errorf("could not retrieve list of security constraints to verify service account %q: %v", serviceAccount, err)
+		}
+	}
+
+	for _, scc := range sccList.Items {
+		if scc.AllowPrivilegedContainer {
+			for _, user := range scc.Users {
+				if strings.Contains(user, serviceAccount) {
+					return nil
+				}
+			}
+		}
+	}
+	return fmt.Errorf("add \"- system:serviceaccount:%s:%s\" to scc privileged\n", ns, serviceAccount)
```

Comment:
see error message construction in the router:
```
		errMsg := "service account %q is not allowed to access the host network on nodes, grant access with oadm policy add-scc-to-user %s -z %s"
		return fmt.Errorf(errMsg, serviceAccount, bootstrappolicy.SecurityContextConstraintsHostNetwork, serviceAccount)
```

give the example command, and use the constant for the scc you are naming